public/static/docs/tutorials/versioning.md

```diff
@@ -245,7 +245,7 @@ That's it! We have a second model and dataset saved and pointers to them
 committed with Git. Let's now look at how DVC can help us go back to the
 previous version if we need to.
 
-## Switching between versions
+## Switching between data or model versions
```

Comment:
same here ... we can switch the whole workspace with git + dvc, right?

Reply:
Yes, using "and" now.